cfn-resources/stream-instance/test/README.md

```diff
@@ -0,0 +1,44 @@
+# Stream Instance
+
+## Impact 
+The following components use this resource and are potentially impacted by any changes. They should also be validated to ensure the changes do not cause a regression.
+ - Stream Connection
```

Comment:
This section is usually defined for components that directly depend on other resources, such as an L3 resource that depends on a specific L1. For this case I do not see any direct implementation dependency between stream connection and stream instance.

Reply:
I included it because deleting a Stream Instance removes all the associated Stream Connections so I thought it was useful to point it out. If this is not the intended use for this section I can remove it

Reply:
I am inclined more for removing as it is more focused on implementation dependency as seen in some other resources, let me know if you find other examples which might be different here.

Reply:
No strong opinion here, and doing a quick search I didn't find any relevant examples. Removed